src/bootstrap/src/lib.rs

```diff
@@ -721,7 +719,7 @@ impl Build {
         if self.config.rustc_parallel && check("rustc_use_parallel_compiler") {
             features.push("rustc_use_parallel_compiler");
         }
-        if self.config.rust_randomize_layout {
+        if self.config.rust_randomize_layout && check("rustc_randomized_layouts") {
```

Comment:
what is `check` for here... it does not exist upstream

Reply:
it does, its a closure a couple lines up. It landed in the same rollup as the `rustc_randomize_layout` config being added hence the discrepancy.